internal/dataplane/synchronizer.go

```diff
@@ -181,7 +183,7 @@ func (p *Synchronizer) startUpdateServer(ctx context.Context) {
 		case <-p.syncTicker.C:
 			if err := p.dataplaneClient.Update(ctx); err != nil {
 				p.logger.Error(err, "could not update kong admin")
-				break
+				continue
```

Comment:
actually `continue` and `break` does the same thing here because `continue` affects `for` but `break` affects `select`.

Reply:
If those are functionally the same, why change it :) ?